src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -27,29 +40,342 @@ const ActorInit En_Door_InitVars = {
     (ActorFunc)EnDoor_Update,
     (ActorFunc)EnDoor_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Door/EnDoor_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Door/EnDoor_Destroy.s")
+/**
+ * Controls which object and display lists to use in a given scene
+ */
+EnDoorInfo D_809FCEA0[] = {
+    { SCENE_HIDAN, 1, OBJECT_HIDAN_OBJECTS },
+    { SCENE_MIZUSIN, 2, OBJECT_MIZU_OBJECTS },
+    { SCENE_HAKADAN, 3, OBJECT_HAKA_DOOR },
+    { SCENE_HAKADANCH, 3, OBJECT_HAKA_DOOR },
+    // KEEP objects should remain last and in this order
+    { -1, 0, OBJECT_GAMEPLAY_KEEP },
+    { -1, 4, OBJECT_GAMEPLAY_FIELD_KEEP },
+};
+
+InitChainEntry D_809FCEC4[] = {
+    ICHAIN_U8(unk_1F, 0, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_STOP),
+};
+
+AnimationHeader* D_809FCECC[] = {
+    0x0400E758,
+    0x0400E5B4,
+    0x04010038,
+    0x0400E6A0,
+};
+
+static u8 sDoorAnimOpenFrames[] = {
+    25,
+    25,
+    25,
+    25,
+};
+
+static u8 sDoorAnimCloseFrames[] = {
+    60,
+    70,
+    60,
+    70,
+};
```

Comment:
```suggestion
static u8 sDoorAnimOpenFrames[] = {
    25,
    25,
    25,
    25
};

static u8 sDoorAnimCloseFrames[] = {
    60,
    70,
    60,
    70
};
```
We probably don't need format.sh putting these all on different lines. Might be true of `D_809FCECC` as well.